examples/sdl3_renderer/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+cmake_minimum_required(VERSION 3.27)
+
+# Project setup
+project(clay_sdl3_renderer C)
+set(CMAKE_C_FLAGS_DEBUG "-g -Wall -Werror")
```

Comment:
Small remark here, which was already present in the other examples: you're replacing the flags here instead of adding them. Ideally, you'd write something like

```cmake
set(CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} -g -Wall -Werror")
```

Reply:
Done